source/base/Benchmarker.ooc

```diff
@@ -0,0 +1,40 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+import Profiler
+import io/FileWriter
+
+Benchmarker: class {
+	_instanceCounter := static 0
```

Comment:
This shouldn't be needed. It's up to the user of this class to supply a unique name.